orca-core/src/test/groovy/com/netflix/spinnaker/orca/pipeline/model/PipelineExecutionImplSpec.groovy

```diff
@@ -76,6 +80,10 @@ class PipelineExecutionImplSpec extends Specification {
         refId = "2"
         outputs["baz"] = "foo-2"
         outputs["qux"] = "qux-2"
+        outputs["myList"] = [[
```

Comment:
worth adding a duplicate to see `distinct` working

Reply:
Done 👍